### DISCUSSION TITLE
error Numpy

### DISCUSSION THREAD

**Discussion by ronbere** [vladmandic/sdnext#1119] [Q&A]
it's important??

19:26:46-019774 WARNING  Package wrong version: numpy 1.23.5 required 1.24.3
19:26:46-021097 INFO     Installing package: numpy==1.24.3
19:26:58-321770 ERROR    Error running pip: install --upgrade numpy==1.24.3

**Comment by ghost** [comment]
I managed to fix it manually by going into the folder and doing the following. 
![Animation](https://github.com/vladmandic/automatic/assets/66656514/8986dc69-fb47-4b74-9d85-77e9ecccb423)

**Comment by TeutonJon78** [comment]
I'm going to guess you're on Windows. I was running into that problem as well. If you check your setup.log file, there will likely be  compaint about access violation for permissions. Vlad has said in other issues it's a python error because of how it does file access.

If you want to make sure it is installed correctly, you can just do:
```
\venv\Scripts\python.exe -m pip uninstall numpy
\venv\Scripts\python.exe -m pip install numpy
```

You can force it to use version 1.24.3 using the sytax the other person showed with ==1.24.3

It will also spit about a bunch of other errors because apparently remb doesn't have all it's requirements listed and there are some conflicts. If you use that extension, there might be issues there.

**Comment by ronbere** [comment]
working but if i restart server, the error stills here

**Comment by Tillerz** [comment]
Isn't a1111 setting up its own environment in venv? I guess if you just do a pip install, it installs stuff into the global system python folder, not into the venv used by a1111?

**Reply by vladmandic** [reply]
true, but it works in global environment as well. venv is preferred just to isolate two different apps using different requirements.
